themes/sgsc/sass/_footer.scss

```diff
@@ -141,6 +141,10 @@ span.wpcf7-not-valid-tip {
 }
 
 // Footer
+.site-footer {
+  border-top: 1px dashed lightgray;
```

Comment:
Can you remove the border from the desktop & tablet size? It's only necessary for mobile width.

Reply:
Removed footer above 690px when footer changes from mob.